WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
I don't care what you think. You started that "logical fallacy" (it isn't one) by forcing me to prove why the _**KINGDOM**_ of Salem should be a KINGDOM. Stop opposing it just because you are convinced it has to be a duchy.